docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
+GitHub. This service is called CIFuzz and can be used to run OSS-Fuzz fuzz targets
+on your project when a pull request is submitted. It has the advantage of
+detecting bugs before they are integrated into your project. This insures that
+less bugs are committed meaning less active security vulnerabilities.
+
+## How it Works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+projects OSS-Fuzz fuzz targets are built and run for
```

Comment:
nit: I think it needs apostrophe after "projects", I'd also skip OSS-Fuzz word here

Reply:
Done.